relational-databases/README.md

```diff
@@ -0,0 +1,13 @@
+
+Relational Databases
```

Comment:
This feels like it could be combined with postgres?

Reply:
What are your thoughts on this when it comes to a topic where there are both specific and general guidelines?

Another good example that's similar to this is testing. We should have a `testing` guide, but we also need a `testing-rspec` and `testing-jest` guide since there are things specific to those frameworks that don't belong in the general guides.

In the case of relational databases, what would we do with the general guidelines if we added a specific `mysql` guide?

#611 goes into this a bit.

One solution besides having separate guides would be to have a single `testing` and `relational-databases` guide that has the framework guidelines as sections.

Reply:
I'll comment there.